### PR TITLE
add new readme

### PR DESCRIPTION
**Pull request by mattlub**
related #21 (trying to clean up after I accidentally messed up eion's previous pr #21 )

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,6 @@
-# nazareth-curriculum
\ No newline at end of file
+# nazareth-curriculum
+Repository for discussions about the content of the nazareth curriculum.
+
+Currently Includes:
+- Minutes from Stop Go Continue
+- Other operational things
```
Comment (rebeccaradding) [inline README.md]: What are these "other operational things?" A "such as..." would be helpful here for future cohorts.

**Comment by m4v15** [comment]
@rebeccaradding added a such as, see what you think?

**Comment by rebeccaradding** [comment]
@mattlub I'm confused, aren't we going to shut down this repo?

**Comment by m4v15** [comment]
He opened this in april, it's just never got merged, shall we just close it? Or merge it anyway for posterity

**Comment by mattlub** [comment]
ok I'm closing it.

**Pull request closed without merging**